ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePageVulnerabilities.tsx

```diff
@@ -165,147 +163,138 @@ function ImagePageVulnerabilities({
-                                <SplitItem>
-                                    <BulkActionsDropdown isDisabled={selectedCves.size === 0}>
-                                        <DropdownItem
-                                            key="bulk-defer-cve"
-                                            component="button"
-                                            onClick={() =>
-                                                showModal({
-                                                    type: 'DEFERRAL',
-                                                    cves: Array.from(selectedCves.values()),
-                                                })
-                                            }
-                                        >
-                                            Defer CVEs
-                                        </DropdownItem>
-                                        <DropdownItem
-                                            key="bulk-mark-false-positive"
-                                            component="button"
-                                            onClick={() =>
-                                                showModal({
-                                                    type: 'FALSE_POSITIVE',
-                                                    cves: Array.from(selectedCves.values()),
-                                                })
-                                            }
-                                        >
-                                            Mark as false positives
-                                        </DropdownItem>
-                                    </BulkActionsDropdown>
-                                </SplitItem>
-                                <Divider
-                                    className="pf-v5-u-px-lg"
-                                    orientation={{ default: 'vertical' }}
-                                />
-                            </>
-                        )}
-                        <SplitItem>
-                            <Pagination
-                                itemCount={totalVulnerabilityCount}
-                                page={page}
-                                perPage={perPage}
-                                onSetPage={(_, newPage) => setPage(newPage)}
-                                onPerPageSelect={(_, newPerPage) => {
-                                    setPerPage(newPerPage);
-                                }}
+    const hiddenSeverities = getHiddenSeverities(querySearchFilter);
+    const hiddenStatuses = getHiddenStatuses(querySearchFilter);
+    const severityCounts = data?.image.imageCVECountBySeverity;
+    const totalVulnerabilityCount = sum([
+        severityCounts?.critical.total ?? 0,
+        severityCounts?.important.total ?? 0,
+        severityCounts?.moderate.total ?? 0,
+        severityCounts?.low.total ?? 0,
+    ]);
+
+    // TODO - Inline `mainContent` into the return statement
```

Comment:
Moving `mainContent` directly into the returned JSX after this change would be more readable. I held off doing this in this PR to avoid an extra large, superficial diff that would make this change more difficult to review.